docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.md

```diff
@@ -0,0 +1,132 @@
+| type | The data type of the field. It specifies the expected data format for the field, enabling proper validation and handling of values during API interactions. |
+| isEditable | Defines if the field's value can be changed. |
+| isCreatable | Defines if the field can be included in requests to provide an initial value during record creation. |
+| validation | Contains validation configurations. Proper validation ensures that the provided data meets the specified criteria. |
+| required | Defines if the field must be provided in API requests. |
+| maxLength/minLength | Defines the minimum/maximum length allowed for the field with a string type. It enforces a lower boundary, ensuring that the field's value meets or exceeds the defined minimum requirement. |
+| max/min | Defines the minimum/maximum value allowed for the field with an integer type. It enforces a lower boundary, ensuring that the field's value meets or exceeds the defined minimum requirement. Optional. |
+
+
+{% info_block infoBox %}
+
+* We recommend setting `isEditable` and `isCreatable` to `false` for fields that serve as identifiers or keys, ensuring their immutability and preserving the integrity of the data model.
+
+* For the fields with numerable values, an integer data type is a non-decimal number between -2147483648 and 2147483647 in 32-bit systems and, in 64-bit systems, between -9223372036854775808 and 9223372036854775807. If you need or anticipate values outside of this range, you can set the value as a string type.
+
+* The Data Exchange API supports the following types for the configured fields:
+  * boolean
+  * integer
+  * string
+  * decimal
+
+{% endinfo_block %}
+
+## Create Data Exchange API endpoints
+
+In this example, we are creating the `/dynamic-data/country` endpoint to operate with data in the `spy_country` table. When following the steps, adjust the data per your requirements:
+
+
+1. In the Back Office, go to **Data Exchange API Configuration**.
+2. On the **Data Exchange API Configuration** page, click **Create Data Exchange API configuration**.
+
+[PASTE SCREENSHOT HERE]
+
+3. Then you need to select a configurable table in the form below. In our case it is `spy_country`:
+
+[PASTE SCREENSHOT HERE]
+
+{% info_block infoBox %}
+
+Note, that you can only select tables that are not excluded from configuring.
+In order to exclude a table from configuring follow [Data Exchange API integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/data-exchange-api-integration.html).
+
+{% endinfo_block %}
+
+After you select a table, you will see a form for configuring the endpoint. Let's fill in the form with the following values:
+
+[PASTE SCREENSHOT HERE]
+
+{% info_block infoBox %}
+
+Note, that you have to enable the endpoint in order to be able to use it. And besides that, you have to provide enable each field you want to use as well.
```

Comment:
1. What endpoint do you mean?
2. What do you mean by using the endpoint?
3. Is it provide or enable in the second sentence?
4. What do you mean by using the field?